### DISCUSSION TITLE
Homebrew Removing Files

### DISCUSSION THREAD

**Discussion by dmitri-labelle** [Homebrew/discussions#3398] [Everyday usage]
Hi there, kind of a newbie with computers, any help would be great. While trying to download something with brew recently it looks like it also went through and updated some of my programs I had installed with homebrew previously. There were many lines like `Removing: /usr/local/Cellar/bdw-gc/8.0.4_1... (69 files, 1.5MB)` and various other files. But I'd like to recover some of those files somehow as I was storing my work in one of them. Is that possible?

**Comment by SMillerDev** [comment]
If you made backups you can restore those.

**Reply by dmitri-labelle** [reply]
Sadly I don't have any super recent back ups
